src/main/kotlin/burrows/apps/example/gif/data/rest/model/RiffsyResponseDto.kt

```diff
@@ -10,4 +10,4 @@ import com.squareup.moshi.Json
  */
 data class RiffsyResponseDto(
   @Json(name = "results") var results: List<ResultDto>? = null,
-  @Json(name = "page") var page: Float? = null)
+  @Json(name = "page") var page: Int? = null)
```

Comment:
@pawelkw Odd, I do not see `page` being used in the response: https://api.riffsy.com/v1/search?key=LIVDSRZULELA&tag=goodluck&limit=10 or the API anymore: https://tenor.com/gifapi#trending.